test/Hpack/UtilSpec.hs

```diff
@@ -97,22 +97,22 @@ spec = do
   describe "expandGlobs" $ around withTempDirectory $ do
     it "accepts simple files" $ \dir -> do
         touch (dir </> "foo.js")
-        expandGlobs dir ["foo.js"] `shouldReturn` ([], ["foo.js"])
+        expandGlobs "" dir ["foo.js"] `shouldReturn` ([], ["foo.js"])
```

Comment:
I think for the purpose of documentation e.g. `"field-name"` is more suitable than `""` or `"XXX"`